changelog/unreleased/issue-4934

```diff
@@ -0,0 +1,9 @@
+Enhancement: Clear removed snapshots, index and pack files from the local cache
+
+Restic did not clear removed snapshots from the cache after the `forget` 
```

Comment:
That remark is too broad. `forget` removes snapshots from the cache, but only on the current host. Please write something like the following:

```
Restic only removed snapshots from the cache on the host that runs the `forget` command.
On other hosts that use the same repository, the old snapshots remained in the cache.

Now, restic automatically clears old snapshots.
```